test/checks/whitespace/OperatorWrapCheckTest.hx

```diff
@@ -78,6 +79,15 @@ abstract OperatorWrapCheckTests(String) to String {
 		}
 	}";
 
+	var TYPEDEF_EXTENSION = "
+	typedef Cord {
```

Comment:
unittests just parse code, they don't compile it, so `Cord` is not really needed here